src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -437,7 +439,7 @@ object DepositHandler {
       .flatten
   }
 
-  def createDepositReceipt(deposit: Deposit, settings: Settings, id: String): DepositReceipt = {
+  def createDepositReceipt(settings: Settings, id: String): DepositReceipt = {
```

Comment:
I recently found a new trick in Scala that would probably nicely go with this method:

```
def createDepositReceipt(settings: Settings, id: String): DepositReceipt = {
  new DepositReceipt {
    val editIRI = new IRI(settings.serviceBaseUrl + "container/" + id)
    val editMediaIri = new IRI(settings.serviceBaseUrl + "media/" + id)
    val stateIri = settings.serviceBaseUrl + "statement/" + id
    setEditIRI(editIRI)
    setLocation(editIRI)
    setEditMediaIRI(editMediaIRI)
    setSwordEditIRI(editIRI)
    setAtomStatementURI(stateIri)
    setPackaging(Collections.singletonList("http://purl.org/net/sword/package/BagIt"))
    setTreatment("[1] unpacking [2] verifying integrity [3] storing persistently")
  }
}
```